### ISSUE TITLE
Unsupported Maven plugin

### ISSUE THREAD

**Issue by rabidpraxis** [honeybadger-io/honeybadger-react-native#33]
We received a support request documenting an issue with an unsupported maven plugin:

> The HoneyBadger plugin appears to be on an old version of Gradle, and we run Gradle 7 in our mobile app. 
> I'm pretty sure Google Play pushes us to be on Gradle 7 in order to upload the app into the app store. 
> The underlying issue appears to be that the Maven plugin isn't supported anymore on Gradle 7:
> https://docs.gradle.org/7.0/userguide/upgrading_version_6.html#removal_of_the_legacy_maven_plugin

**Comment by jwigal** [comment]
Still looking into this, but here's the specific error I'm seeing when compiling under Android:

```


FAILURE: Build completed with 2 failures.

1: Task failed with an exception.
-----------
* Where:
Build file '/Users/jeff/code/assignr-mobile/node_modules/@honeybadger-io/react-native/android/build.gradle' line: 23

* What went wrong:
A problem occurred evaluating project ':@honeybadger-io_react-native'.
> Plugin with id 'maven' not found.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
==============================================================================

2: Task failed with an exception.
-----------
* What went wrong:
A problem occurred configuring project ':@honeybadger-io_react-native'.
> compileSdkVersion is not specified. Please add it to build.gradle

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
==============================================================================

* Get more help at https://help.gradle.org

BUILD FAILED in 3s
```

**Comment by andreybutov** [comment]
@rabidpraxis @jwigal This is fixed in 0.0.11.
